ui/src/modules/Settings/Credentials/Sections/Registry/Form.tsx

```diff
@@ -188,19 +225,43 @@ const FormRegistry = ({ onFinish }: Props) => {
   const renderForm = () => (
     <Styled.Form onSubmit={handleSubmit(onSubmit)}>
       <Text.h5 color="dark">
-        Fill in the fields below with your information:
+        Fill in the fields below with your {registryName} information:
       </Text.h5>
       <Styled.Fields>
         <Form.Input
           ref={register({ required: true })}
           name="name"
           label="Type a name for Registry"
         />
-        <Form.Input
-          ref={register({ required: true })}
-          name="address"
-          label="Enter the registry url"
-        />
+        {registryType !== 'DOCKER_HUB' && (
+          <>
+            <Form.Input
+              ref={register({
+                required: true,
+                validate: {
+                  methodValidate: (value: string) => {
+                    if (value === 'https://' || value === 'http://') {
+                      return false;
+                    } else if (
+                      value.includes('https://') ||
+                      value.includes('http://')
+                    ) {
+                      return true;
+                    }
+                    return false;
+                  }
```

Comment:
`import endsWith from 'lodash/endsWith'`

```suggestion
   return (!endsWith(value, 'https://') && !endsWith(value, 'http://')) || !isEmpty(value);
```

I made this in my mind, so maybe the condition won't fit the problem.

Reply:
this condition does not check whether the string contains https:// or http://.
we need to check: 
1st if string is just https or http.
2nd if the string contains https or http and more text
3rd if the string exists, but does not contain https or http, returns false